tools/test/topology/test-multi-playback.m4

```diff
@@ -0,0 +1,114 @@
+#
+# Topology for pass through pipeline
+#
+
+# Include topology builder
+include(`pipeline.m4')
+include(`dai.m4')
+include(`ssp.m4')
+include(`utils.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include Apollolake DSP configuration
+include(`platform/intel/bxt.m4')
+
+DEBUG_START
+
+#
+# Machine Specific Config - !! MUST BE SET TO MATCH TEST MACHINE DRIVER !!
+#
+# TEST_PIPE_NAME - Pipe name
+# TEST_DAI_LINK_NAME - BE DAI link name e.g. "NoCodec"
+# TEST_DAI_PORT	- SSP port number e.g. 2
+# TEST_DAI_FORMAT - SSP data format e.g s16le
+# TEST_PIPE_FORMAT - Pipeline format e.g. s16le
+# TEST_SSP_MCLK - SSP MCLK in Hz
+# TEST_SSP_BCLK - SSP BCLK in Hz
+# TEST_SSP_PHY_BITS - SSP physical slot size
+# TEST_SSP_DATA_BITS - SSP data slot size
+# TEST_SSP_MODE - SSP mode e.g. I2S, LEFT_J, DSP_A and DSP_B
+#
+
+# Apply a non-trivial filter blob IIR and FIR tests. TODO: Note that the
+# PIPELINE_FILTERx notation will be updated in future for better flexibility.
+define(PIPELINE_FILTER1, ifelse(TEST_PIPE_NAME, `eq-iir', `eq_iir_coef_loudness.m4'))
```

Comment:
this should be crossover? otherwise TEST_PIPE_NAME will be inserted as eq-iir below no?

Reply:
These lines are copied from test-playback.m4 and I think this takes no effect here. I could remove then.
Referenced from:
https://www.gnu.org/software/m4/manual/m4-1.4.15/html_node/Ifelse.html

define(argument-1, ifelse(string-1, string-2, equal, [not-equal]))
when "string-1" and "string-2" are equal, ifelse() will expand to "equal", then it becomes define(argument-1, equal).

In our case, TEST_PIPE_NAME will only be `crossover', so PIPELINE_FILTER1 defines nothing. (However even in test-playback.m4 PIPELINE_FILTER1 doesn't appear anywhere else. I suspect they're dummy lines?)